poh-bundle-win/oh.ps1

```diff
@@ -492,7 +491,7 @@ function import_database {
 	# Create OH database structure
 	Write-Host "Importing database schema $DB_CREATE_SQL..."
 	
-	cd "$OH_PATH\$SQL_DIR"
+	cd "./$SQL_DIR"
 
     $SQLCOMMAND=@"
    --local-infile=1 -u root -p$MYSQL_ROOT_PW -h $MYSQL_SERVER --port=$MYSQL_PORT --protocol=tcp $DATABASE_NAME -e "source $OH_PATH\$SQL_DIR\$DB_CREATE_SQL"
```

Comment:
full path we already probed doesn't work, since we are already in the folder is enough `-e "source $DB_CREATE_SQL"`